ComponentViewer.Docs/Pages/Index.razor

```diff
@@ -234,7 +234,8 @@
         new("MudPopup", "A mobile friendly popup content for several situations."),
         new("MudRangeSlider", "A slider with range capabilities, set upper and lower values."),
         new("MudScrollbar", "Handle all or selected scrollbar styles with a Mud component."),
-        new("MudSpeedDial", "An expandable fab component."),
+        new("MudWheel", "Smoothly changes values in a wheel within defined ItemCollection."),
```

Comment:
Component should alphabetically order, we have a little mistake here. And we should add SignaturePad also in MainLayout.razor to show navigate component with navbar.